Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    background: var(--white);
+    height: 100px;
+}
+
+header div {
+    text-align: center;
+    padding-top: 1rem;
+}
+
+h1 {
+    font-family: var(--header-font);
+    margin: 0;
+}
+
+section {
+    background: var(--db);
+    height: 1000px;
```

Comment:
Again, avoid these explicit heights. A great exercise for refactoring is to open up Devtools and start unchecking styles to discover what is really needed/necessary.

Reply:
Good practice, thanks. Removed explicit heights. I left min-height, I hope this is okay